acquia/cli
# CLI-799: Support multiple dbs

**Motivation**

Support multiple DBs in IDE

**Proposed changes**

Since IDEs have a passwordless root MySQL user like Lando, we can use the same business logic

**Testing steps**


1. Follow the [contribution guide](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. Run `pull:db` with multiple databases at once, including for databases that don't exist in the IDE yet, to ensure they are created and imported.
